### ISSUE TITLE
Matrix 403 errors

### ISSUE THREAD

**Issue by klassare** [airgap-it/beacon-sdk#316]
**Describe the bug (current behavior)**
403 errors on the join request, resulting in no communication between wallet and dapp.

**Screenshots and/or logs**
On the wallet side I got these errors 8 days ago
![screenshot_45](https://user-images.githubusercontent.com/31806893/155578479-67aecd24-6df3-49c0-bd08-fdbc57df1d8e.jpg)

On wallet side from user report today.
![maim-region-20220224-190824](https://user-images.githubusercontent.com/31806893/155578720-1c14359c-ea0d-48a8-b084-ef2dde64dcb2.png)

Same user also have 403 errors on the dapp side
![maim-region-20220224-195039](https://user-images.githubusercontent.com/31806893/155579153-caa1ca40-027c-48dd-916f-652fe1a04526.png)
![maim-region-20220224-195206](https://user-images.githubusercontent.com/31806893/155579165-c66549a0-96fa-4335-aa94-bc345ea25c61.png)

**Additional context**
objkt.com and Kukai used in both cases

**Comment by klassare** [comment]
Asked the user to complement with response body

dapp side
![maim-region-20220224-202106](https://user-images.githubusercontent.com/31806893/155583794-cf4c1cfb-908e-4de6-99b6-284e76366d39.png)

wallet side
![maim-region-20220224-202253](https://user-images.githubusercontent.com/31806893/155583812-178cd457-6978-4f7f-b813-e68a46eff78a.png)

**Comment by AndreasGassmann** [comment]
We investigated where the "Invalid password" error could come from. The most likely issue is the system time of the client being out of sync with the server time. We identified one case where this could easily happen if the local time is ahead of the server time. We have addressed this problem on the node and will roll it out over the next few days.

In the medium to long term, we will add an additional endpoint to the beacon-node where the client can read the server time and use it in the signature instead of its local time.
